ansible/roles/mongod/templates/repset_init.j2

```diff
@@ -1,6 +1,6 @@
 rs.initiate({ 
   "_id": "mqperf",
-  "members": [ { "_id": 0, host: "{{ hostvars[(groups['mongo'] | first)].ec2_private_ip_address }}" } ]
+  "members": [ { "_id": 0, host: "{{ hostvars[mongo_primary].ec2_private_ip_address }}" } ]
```

Comment:
good idea 👍  
Let me extract `mongo_secondaries` as well (will extend to this PR) :)